data-prepper-plugins/service-map-stateful/src/main/java/com/amazon/dataprepper/plugins/processor/ServiceMapStatefulProcessor.java

```diff
@@ -24,7 +25,7 @@
 import java.util.stream.Stream;
 
 @DataPrepperPlugin(name = "service_map_stateful", type = PluginType.PROCESSOR)
-public class ServiceMapStatefulProcessor implements Processor<Record<ExportTraceServiceRequest>, Record<String>> {
+public class ServiceMapStatefulProcessor extends AbstractPrepper<Record<ExportTraceServiceRequest>, Record<String>> {
```

Comment:
Do you want to update the name to `ServiceMapStatefulPrepper` - does not like the sounding but just asking if you intent to ?

Reply:
Sure, I'll do that